src/TourManager/TourManager.Service.Abstraction/ITourService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+using TourManager.Service.Model;
+using System.Collections.Generic;
+
+namespace TourManager.Service.Abstraction
+{
+    public interface ITourService
```

Comment:
we also need IClientService, IPropertyService I guess

Reply:
yep